mods/ts/rules/defaults.yaml

```diff
@@ -101,7 +107,7 @@
 
 ^Cloakable:
 	Cloak@EXTERNALCLOAK:
-		RequiresCondition: cloakgenerator || crate-cloak
+		RequiresCondition: cloakgenerator + crate-cloak
```

Comment:
I would strongly prefer that we implicitly converted isolated ints to bools in this case and then used the normal boolean operators.  Forcing math operators over boolean operators in this case is a serious step backwards in legibility.